src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -175,10 +221,15 @@ public PersonListPanel getPersonListPanel() {
     private CommandResult executeCommand(String commandText) throws CommandException, ParseException {
         try {
             CommandResult commandResult = logic.execute(commandText);
-            logger.info("Result: " + commandResult.getFeedbackToUser()); // used to log the result, not actually
-            // used to display
+            logger.info("Result: " + commandResult.getFeedbackToUser());
             resultDisplay.setFeedbackToUser(commandResult.getFeedbackToUser());
 
+            if (commandText.trim().toLowerCase().startsWith("view")) {
+                handleViewCommand(commandText);
+            } else if (commandText.trim().toLowerCase().equals("close")) {
+                handleCloseCommand();
+            }
+
```

Comment:
Perhaps to align with the same levels of abstraction, the boolean evaluations (trim().toLowerCase...) can be moved to command result (similar to how below it is commandResult.isShowHelp); and abstracted as isShowView